### DISCUSSION TITLE
Unable to Change Email on GitHub Due to Sponsorship Profile Issue

### DISCUSSION THREAD

**Discussion by mdjamilkashemporosh** [community/community#145331] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, I am currently trying to change the email associated with my GitHub profile. However, when I attempt to delete the old email, I receive an error message stating that the email is connected to my sponsorship profile. Since my sponsorship profile is in a pending state, I am unable to edit or delete it. Is there any way to resolve this, or am I missing something in the process?

**Comment by Awwze** [comment, marked as answer]
Hi mdjamilkashemporosh,

It seems like the issue is related to the email being tied to your sponsorship profile. When a sponsorship profile is in a pending state, certain changes, such as editing or deleting the associated email, might be restricted.

Here are a few steps you can try to resolve this:

Check the Sponsorship Status:

If the sponsorship profile is pending approval, you may need to wait for the approval process to complete before making changes.
You can also reach out to GitHub Support to confirm the current status of your sponsorship profile.
Update the Email in the Sponsorship Profile:

Navigate to your sponsorship profile settings and attempt to update the email address directly there.
Contact GitHub Support:

If neither of the above options works, contacting [GitHub Support](https://support.github.com/contact) would be the best way to resolve this issue. Be sure to explain the situation and provide details about the error message you’re receiving. They can assist with clearing any restrictions or provide guidance on next steps.
Let us know if you find a solution, or if there are any updates!

**Reply by mdjamilkashemporosh** [reply]
Thanks
